### PR TITLE
Add Logs to doc website repo.

### PR DESCRIPTION
**Pull request by carolxob**
Signed-off-by: carolxob <carolxob@amazon.com>

### Description
Add Logs to doc website repo.

### Issues Addressed
Addresses #1267. 

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by carolxob** [comment]
@dlvenable This PR is ready for tech review.

**Review by dlvenable — APPROVED** [review]

**Comment by carolxob** [comment]
@cmanning09 This PR is ready for tech review.

**Comment by carolxob** [comment]
@arjunnambiartc This is ready for tech review.

**Comment by carolxob** [comment]
@asifsmohammed This is ready for tech review.

**Review by asifsmohammed — APPROVED** [review]
LGTM

File: _data-prepper/logs.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+---
+
+#Log4j configuration
+
+The following describes how Data Prepper performs logging.
+
+# Logging 
+Data Prepper uses [SLF4J](http://www.slf4j.org/) with a [Log4j 2 binding](http://logging.apache.org/log4j/2.x/log4j-slf4j-impl/). 
+
+For Data Prepper 2.0 or above, Log4j 2 configuration file can be found and edited in `config/log4j2.properties` in the application's home directory. Default properties for Log4j 2 can be found in `log4j2-rolling.properties` in the *shared-config* directory.
```
Comment (cwillum) [inline _data-prepper/logs.md:14]: "For Data Prepper versions 2.0 and higher ..." (See "above" in TERMS.md)
"... the Log4j 2 configuration file can be ..."

File: _data-prepper/logs.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+For Data Prepper before version 2.0, Log4j 2 configuration can be overridden by setting "log4j.configurationFile" system property when running Data Prepper. Default properties for Log4j 2 can be found in `log4j2.properties` in the *shared-config* directory.
```
Comment (cwillum) [inline _data-prepper/logs.md:16]: "... by setting the "log4j.configurationFile" system property ..." 
I believe a system property is set in a configuration file and we would express this as code. If that's right, then it should be set as code: `log4j.configurationFile`.
Also, if this is a property, is it Boolean? In other words, would it be accurate to say:
"... by setting the `log4j.configurationFile` system property to `true`."?

**Review by cwillum — COMMENTED** [review]
Let me know if you have questions with any of these comments.

File: _data-prepper/logs.md (hunk 2 above)
Comment (carolxob) [reply _data-prepper/logs.md]: @oeyh Can you confirm that saying "... by setting the `log4j.configurationFile` system property to `true`" is accurate here?

Comment (oeyh) [reply _data-prepper/logs.md]: It's not like that. Please refer to the example command following the above text:

```
java "-Dlog4j.configurationFile=config/custom-log4j2.properties" -jar data-prepper-core-$VERSION.jar pipelines.yaml data-prepper-config.yaml
```

This is how `log4j.configurationFile` property is set when running data prepper, with `-D{property_name}={property_value}`. The value here is a path to the log4j2 config file.

Comment (carolxob) [reply _data-prepper/logs.md:17]: @oeyh Would this work? "The following command is how the `log4j.configurationFile` property is set when running Data Prepper.  This property uses `-D{property_name}={property_value}` as the value, which is a path to the Log4j2 configuration file."

Comment (carolxob) [reply _data-prepper/logs.md:17]: @chenqi0805 Would this work? "The following command is how the log4j.configurationFile property is set when running Data Prepper. This property uses -D{property_name}={property_value} as the value, which is a path to the Log4j2 configuration file."

Comment (chenqi0805) [reply _data-prepper/logs.md:17]: How about making it more concise by 
```
...can be overridden by setting system property "-Dlog4j.configurationFile={property_value}" when running Data Prepper, where {property_value} is a path to the Log4j2 configuration file.
```

File: _data-prepper/logs.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+---
+layout: default
+title: Log4j configuration
+nav_order: 12
+---
+
+# Log4j configuration
+
+This section provides the details for configuring Log4j.
```
Comment (natebower) [inline _data-prepper/logs.md:9]:
```suggestion
This section provides information about configuring Log4j. 
```

File: _data-prepper/logs.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+This section provides the details for configuring Log4j. 
+
+## Logging 
+
+The following describes how Data Prepper performs logging. Data Prepper uses [SLF4J](http://www.slf4j.org/) with a [Log4j 2 binding](http://logging.apache.org/log4j/2.x/log4j-slf4j-impl/). 
+
+<!--- Editorial: Is it safe to remove this first introductory sentence? I'm not sure if it's necessary or helpful to users.--->
+
+For Data Prepper version 2.0 and later, the Log4j 2 configuration file can be found and edited in `config/log4j2.properties` in the application's home directory. The default properties for Log4j 2 can be found in `log4j2-rolling.properties` in the *shared-config* directory.
```
Comment (natebower) [inline _data-prepper/logs.md:17]:
```suggestion
For Data Prepper versions 2.0 and later, the Log4j 2 configuration file can be found and edited in `config/log4j2.properties` in the application's home directory. The default properties for Log4j 2 can be found in `log4j2-rolling.properties` in the *shared-config* directory.
```

File: _data-prepper/logs.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+For Data Prepper versions before 2.0, Log4j 2 configuration can be overridden by setting "log4j.configurationFile" system property when running Data Prepper. Default properties for Log4j 2 can be found in `log4j2.properties` in the *shared-config* directory.
```
Comment (natebower) [inline _data-prepper/logs.md:19]:
```suggestion
For Data Prepper versions before 2.0, the Log4j 2 configuration can be overridden by setting the "log4j.configurationFile" system property when running Data Prepper. Default properties for Log4j 2 can be found in `log4j2.properties` in the *shared-config* directory. 
```

Comment (natebower) [reply _data-prepper/logs.md:19]: Delete the second sentence, as it appears in the preceding paragraph. However, note that the text in code format is different in each case.

File: _data-prepper/logs.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+<!--- Editorial: Is it safe to remove this first introductory sentence? I'm not sure if it's necessary or helpful to users.--->
+
+For Data Prepper version 2.0 and later, the Log4j 2 configuration file can be found and edited in `config/log4j2.properties` in the application's home directory. The default properties for Log4j 2 can be found in `log4j2-rolling.properties` in the *shared-config* directory.
+
+For Data Prepper versions before 2.0, Log4j 2 configuration can be overridden by setting "log4j.configurationFile" system property when running Data Prepper. Default properties for Log4j 2 can be found in `log4j2.properties` in the *shared-config* directory. 
+
+### Example
+
+When running Data Prepper, the following command can be overridden by setting system property `-Dlog4j.configurationFile={property_value}`, where `{property_value}` is a path to the Log4j2 configuration file.
```
Comment (natebower) [inline _data-prepper/logs.md:23]:
```suggestion
When running Data Prepper, the following command can be overridden by setting the system property `-Dlog4j.configurationFile={property_value}`, where `{property_value}` is a path to the Log4j2 configuration file:
```

Comment (natebower) [reply _data-prepper/logs.md:23]: Preceding this, there is a space between "Log4j" and "2".

File: _data-prepper/logs.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+
+<!--- Editorial: Do we need a colon at the end?--->
+
+```
+java "-Dlog4j.configurationFile=config/custom-log4j2.properties" -jar data-prepper-core-$VERSION.jar pipelines.yaml data-prepper-config.yaml
+```
+
+See the [Log4j 2 configuration docs](https://logging.apache.org/log4j/2.x/manual/configuration.html) for more information on Log4j 2 configurations.
```
Comment (natebower) [inline _data-prepper/logs.md:31]:
```suggestion
See the [Log4j 2 configuration documentation](https://logging.apache.org/log4j/2.x/manual/configuration.html) for more information about Log4j 2 configuration.
```

File: _data-prepper/logs.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+
+For Data Prepper version 2.0 and later, the Log4j 2 configuration file can be found and edited in `config/log4j2.properties` in the application's home directory. The default properties for Log4j 2 can be found in `log4j2-rolling.properties` in the *shared-config* directory.
+
+For Data Prepper versions before 2.0, Log4j 2 configuration can be overridden by setting "log4j.configurationFile" system property when running Data Prepper. Default properties for Log4j 2 can be found in `log4j2.properties` in the *shared-config* directory. 
+
+### Example
+
+When running Data Prepper, the following command can be overridden by setting system property `-Dlog4j.configurationFile={property_value}`, where `{property_value}` is a path to the Log4j2 configuration file.
+
+<!--- Editorial: Do we need a colon at the end?--->
```
Comment (natebower) [inline _data-prepper/logs.md:25]: Yes, added to suggestion.

File: _data-prepper/logs.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+
+# Log4j configuration
+
+This section provides the details for configuring Log4j. 
+
+## Logging 
+
+The following describes how Data Prepper performs logging. Data Prepper uses [SLF4J](http://www.slf4j.org/) with a [Log4j 2 binding](http://logging.apache.org/log4j/2.x/log4j-slf4j-impl/). 
+
+<!--- Editorial: Is it safe to remove this first introductory sentence? I'm not sure if it's necessary or helpful to users.--->
```
Comment (natebower) [inline _data-prepper/logs.md:15]: This seems like something we should ask a SME. Is it necessary to understanding what follows? If yes, then we should include it. If no, you can make a decision regarding whether or not to include it based on its potential helpfulness to the reader as complementary information.

**Review by natebower — COMMENTED** [review]
@carolxob Please see my comments and changes and let me know if you have any questions. Thanks!

File: _data-prepper/logs.md (hunk 9 above)
Comment (carolxob) [reply _data-prepper/logs.md:15]: @natebower Oh, I was specifically referring only to the sentence: "The following describes how Data Prepper performs logging." The subsequent text seems to explain it. Maybe we can change it to something like "Data Prepper performs logging by using [SLF4J](http://www.slf4j.org/) with a [Log4j 2 binding](http://logging.apache.org/log4j/2.x/log4j-slf4j-impl/)." ?

**Review by Naarcha-AWS — APPROVED** [review]

**Review by ariamarble — APPROVED** [review]

**Pull request merged**